CometServer/Modules/10-25/ExchangeFileExportApi.cs

```diff
@@ -117,28 +117,27 @@ public override void AddRoutes(IEndpointRouteBuilder app)
                     return;
                 }
 
-                if (!req.HttpContext.User.Identity.IsAuthenticated)
+                if (!(req.HttpContext.User.Identity?.IsAuthenticated ?? false))
```

Comment:
can we have the situation that `Identity` is null?

Reply:
i see that the integration tests are failing, can you please check the GitHub action?

Reply:
Due to transitive dependency to NewtonSoft 11.0.1 in Hangfire. Fixed.

Reply:
> can we have the situation that `Identity` is null?

Normally not, but this was a recommendation from ReSharper.